### DISCUSSION TITLE
CMAKE New Version

### DISCUSSION THREAD

**Discussion by jasr93** [xpack-dev-tools/cmake-xpack#5] [General]
Hello,
The current latest version of xpack cmake is 3.23.5-1. However, the latest official release of cmake is 3.26.4.
I've seen several projects that require at least version 3.24.
Is there a timeline on when a new version of xpack cmake will be released?
Thanks

**Comment by ilg-ul** [comment]
There is no clear schedule, but usually July is a good moment to update some of the binary tools, so I'll happen probably during the next few weeks.

**Comment by ilg-ul** [comment]
Better keep it open to remind me of the new release.

**Comment by ilg-ul** [comment]
3.24.4-1 is now released. 3.25.3-1 will come soon.

**Reply by ilg-ul** [reply]
3.25.3-1 is out.
